.github/workflows/all_tests_cron.yml

```diff
@@ -1,11 +1,15 @@
-name: test master cron
+name: All tests cron
 on:
   schedule:
-    - cron: '0 5 * * *' 
+    - cron: "0 5 * * *"
+    - cron: "30 * * * 6,0"
 jobs:
-  dependencies:
-    name: Get Dependencies
```

Comment:
I originally had a seperate job that installed deps, but it doesn't carry over the installation into the matrix. This slows down the job a bit, but I haven't found a way around it.